QuizWhois.Api/Controllers/QuizController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using QuizWhois.Common.Models;
+using QuizWhois.Domain.Services.Interfaces;
+
+namespace QuizWhois.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class QuizController
+    {
+
+        private readonly IQuizService _quizService;
+
+        public QuizController(IQuizService quizService)
+        {
+            _quizService = quizService;           
+        }
+
+        // [Authorize]
+        [HttpPost("AddToSet")]
```

Comment:
why not to use something like HttpPost("{quizId}")?

Reply:
But i would consider [HttpPost("AddToSet/{quizId}")]

Reply:
@sidgrouse Something like that? 
![image](https://user-images.githubusercontent.com/63876987/137540996-1da6e33f-256e-44b9-b36f-3343986079d0.png)